src/App.js

```diff
@@ -1,3 +1,4 @@
+import { EntryList } from "./EntryList";
```

Comment:
You just have imported the EntryList component
you need to render it inside App component

Reply:
![image](https://user-images.githubusercontent.com/60723932/173144652-8e591f38-91c2-4b40-bdbd-86bdda3a5fbe.png)

Reply:
![image](https://user-images.githubusercontent.com/60723932/173144733-7814bfa4-425d-40eb-b264-a871d201afd4.png)

Reply:
Did you export EntryList ?

Reply:
![image](https://user-images.githubusercontent.com/60723932/173146805-929f12ce-79f6-4dad-b132-34220ce90803.png)